apps/qtmozembed/declarativewebpage.cpp

```diff
@@ -203,15 +200,7 @@ void DeclarativeWebPage::restoreHistory() {
 
 void DeclarativeWebPage::setContentLoaded()
```

Comment:
Can it be dropped as empty and unused?

Reply:
Looks like it should be dropped, otherwise LGTM.

Reply:
Will drop now.

Reply:
Oops. Accidentally left completely empty method there but hey that's why we have reviews.

Reply:
Fixed now.